cmd/tailscaled/install_darwin.go

```diff
@@ -66,7 +72,7 @@ func uninstallSystemDaemonDarwin(args []string) (ret error) {
 			fmt.Printf("launchctl stop com.tailscale.tailscaled: %v, %s\n", err, out)
 			ret = err
 		}
-		out, err = exec.Command("launchctl", "unload", sysPlist).CombinedOutput()
+		out, err = exec.Command("launchctl", "unload", "-w", sysPlist).CombinedOutput()
```

Comment:
Do we know when launctl added the `-w` flag? macOS releases back to 10.13 are supported.

Reply:
I haven't found a changelog for launctl, so I'm not sure if the `-w` option is available in 10.13, unfortunately I don't currently have a 10.13 system.

So do I need to remove this option? Or do some other work?

Reply:
This appears to exist at least as of Sept. 2016, per the source code for `launchctl`:
https://github.com/apple-opensource/launchd/blob/master/support/launchctl.c#L2649-L2651

That source release is `launchd 842.92.1`, which was in macOS 10.9.5, before 10.13.